### PR TITLE
feat: add binfmt_misc

### PR DESCRIPTION
**Pull request by sergelogvinov**
Add binfmt-misc kernel support.

https://github.com/siderolabs/pkgs/pull/804

### CODE REVIEW

File: storage/binfmt-misc/pkg.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: binfmt-misc
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  # The pkgs version for a particular release of Talos as defined in
+  # https://github.com/siderolabs/talos/blob/<talos version>/pkg/machinery/gendata/data/pkgs
+  - image: "{{ .PKGS_PREFIX }}/binfmt-misc-pkg:{{ .BUILD_ARG_PKGS }}"
```
Comment (smira) [inline storage/binfmt-misc/pkg.yaml:8]: can we make it pull from `kernel` pkg

File: Makefile [hunk 2]
```diff
@@ -42,6 +42,7 @@ space = $(empty) $(empty)
 TARGETS = \
 		amdgpu-firmware \
 		amd-ucode \
+		binfmt-misc \
```
Comment (smira) [inline Makefile:45]: we also need to add it to `README` please

File: storage/binfmt-misc/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# binfmt misc
```
Comment (smira) [inline storage/binfmt-misc/README.md:1]: I don't think this is `storage` (I mean folder)

Comment (sergelogvinov) [reply storage/binfmt-misc/README.md:1]: I am not sure the best place for it. May be we will create the folder like `modules`?

Comment (smira) [reply storage/binfmt-misc/README.md]: I moved it to `misc`

**Review by frezbo — APPROVED** [review]

**Pull request merged**